### PR TITLE
ui: add Rangefeed catchup charts

### LINKED ISSUE #97240: ui: display changefeed catchup scan metrics
When changefeeds restart, they have to run a catchup scan to find recently changed rows. These catchup scans can be disruptive for large tables. We should include charts for these in the DB Console under Changefeeds. Specifically these metrics:

* `distsender.rangefeed.catchup_ranges`: number of catchup scans on individual ranges (graph gauge for cluster)
* `kv.rangefeed.catchup_scan_nanos`: time spent doing catchup scans (graph rate per node)

Jira issue: CRDB-24587

### PR DESCRIPTION
**Pull request by koorosh**
This change introduces two new charts on Changefeeds dashboard: "Ranges in catchup mode" and "RangeFeed catchup scans duration" that allow to detect disruptive scans after changefeeds restart.

Release note (ui change): added "Ranges in catchup mode" and "RangeFeed catchup scans duration" charts on Changefeeds dashboard.

Resolves: #97240

Screenshot 2023-05-26 at 00 02 50

### CODE REVIEW

**Review by maryliag — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/103927#-:-NWNKin3BSv28TtRR_IK:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @erikgrinaker)

File: pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx [hunk 1]
```diff
@@ -92,5 +97,38 @@ export default function (props: GraphDashboardProps) {
...
+      title="Ranges in catchup mode"
+      isKvGraph={false}
+      sources={storeSources}
+      tooltip="Counts the number of ranges with an active rangefeed that are performing catchup scan"
+    >
+      <Axis units={AxisUnits.Count} label="ranges">
+        <Metric
+          name="cr.node.distsender.rangefeed.catchup_ranges"
+          title="Ranges"
+          aggregateMax
```
Comment (erikgrinaker) [inline pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx:111]: Shouldn't this be sum? We want the total number of ranges across all nodes.

**Review by erikgrinaker — COMMENTED** [review]
Just to make sure I don't misunderstand: we're showing a single cluster-wide gauge for `catchup_ranges`, and a per-store rate for `catchup_scan_nanos`, right?

**Review by koorosh — COMMENTED** [review]
That's correct.

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/103927#-:-NWRovM2DhN4tu7uwlFj:bmwtcq4)* status: :shipit: complete! 0 of 0 LGTMs obtained (and 1 stale) (waiting on @erikgrinaker)

___
*[`pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx` line 105 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/103927#gh-1206897873:-NWRsI4eEaNxoql_pEgC:b-896fix) ([raw file](https://github.com/cockroachdb/cockroach/blob/9cd0dcb68ddc2be780636833d5b5799ce2c44099/pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx#L105)):*
<details><summary><i>Previously, erikgrinaker (Erik Grinaker) wrote…</i></summary><blockquote>

```suggestion
      tooltip="Total number of ranges with an active rangefeed that are performing catchup scan"
```
</blockquote></details>

Done.
___
*[`pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx` line 111 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/103927#gh-1206899475:-NWRs6yR9xZxFF_piQqZ:boqn516) ([raw file](https://github.com/cockroachdb/cockroach/blob/9cd0dcb68ddc2be780636833d5b5799ce2c44099/pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/changefeeds.tsx#L111)):*
<details><summary><i>Previously, erikgrinaker (Erik Grinaker) wrote…</i></summary><blockquote>

Shouldn't this be sum? We want the total number of ranges across all nodes.
</blockquote></details>

Right, should be `sum`. Changed.

**Review by erikgrinaker — APPROVED** [review]
LGTM, thanks!

**Pull request merged**